lib/puppet/provider/grafana.rb

```diff
@@ -1,17 +1,3 @@
-#    Copyright 2015 Mirantis, Inc.
```

Comment:
I am not sure if we can remove the copyright. Any apache-2 license experts around?

Reply:
I could also put the copyright back in, but let's see if anyone knows.